runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -7262,6 +7262,11 @@ static void jitHookReleaseCodeGCCycleEnd(J9HookInterface **hook, UDATA eventNum,
    {
    MM_GCCycleEndEvent *event = (MM_GCCycleEndEvent *)eventData;
    OMR_VMThread *omrVMThread = event->omrVMThread;
+
+	if (OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL == (event->cycleType & OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL)) {
+	   /* for scavenger abort case, don't need to code cache reclaim here, the following glocal gc would do the reclamation */
+	   return;
+   }
```

Comment:
I believe you're using tabs here, but you'll need to use spaces. Also the brace brackets are not consistent with the code format in the rest of the file.

Reply:
give a hint of why

We skip the reclamation in the case of an unsecessfully completed GC cycle (such as aborted Scavenge) since it does not have full object liveness info.